workers/data_refinery_workers/downloaders/sra.py

```diff
@@ -202,21 +285,29 @@ def download_sra(job_id: int) -> None:
     job = utils.start_job(job_id)
     file_assocs = DownloaderJobOriginalFileAssociation.objects.filter(downloader_job=job)
 
+    original_file = file_assocs[0].original_file
+    sample = original_file.samples.first()
+    if _has_unmated_reads(sample.accession_code):
+        original_files = _replace_dotsra_with_fastq_files(sample, job, original_file)
+    else:
+        # _replace_dotsra_with_fastq_files returns a list of
+        # OriginalFiles so turn the queryset of
+        # DownloaderJobOriginalFileAssociations into a list of
+        # OriginalFiles to be consistent.
+        original_files = [assoc.original_file for assoc in file_assocs]
```

Comment:
Why not initialize it to `job.original_files`? Seems like it might help to remove `file_assocs` above.